scripts/quests/otherAreas/For_the_Birds.lua

```diff
@@ -0,0 +1,169 @@
+                        return quest:event(15, 0, xi.item.ARNICA_ROOT)
+                    elseif prog == 1 and not player:hasKeyItem(xi.ki.GLITTERING_FRAGMENT) then
+                        return quest:progressEvent(16, 0, xi.ki.GLITTERING_FRAGMENT)
+                    elseif prog == 2 then
+                        return quest:event(17)
+                    elseif prog == 3 then
+                        return quest:progressEvent(18)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [16] = function(player, csid, option, npc)
+                    npcUtil.giveKeyItem(player, xi.ki.GLITTERING_FRAGMENT)
+                    quest:setVar(player, 'Prog', 2)
+                end,
+
+                [18] = function(player, csid, option, npc)
+                    if quest:complete(player) then
+                        player:delKeyItem(xi.ki.GLITTERING_FRAGMENT)
+                        xi.quest.setMustZone(player, xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.BETTER_THE_DEMON_YOU_KNOW)
+                    end
+                end,
+            },
+        },
+
+        [xi.zone.CASTLE_OZTROJA] =
+        {
+            ['Daa_Bola_the_Seer'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 0 then
+                        return quest:event(86)
+                    end
+                end,
+
+                onTrade = function(player, npc, trade)
+                    if
+                        quest:getVar(player, 'Prog') == 0 and
+                        npcUtil.tradeHasExactly(trade, xi.item.ARNICA_ROOT)
+                    then
+                        return quest:progressEvent(87, 0, xi.item.ARNICA_ROOT)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [87] = function(player, csid, option, npc)
+                    player:tradeComplete()
+                    quest:setVar(player, 'Prog', 1)
+                end,
+            },
+        },
+
+        [xi.zone.BEADEAUX] =
+        {
+            ['GeFhu_Yagudoeye'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        quest:getVar(player, 'Prog') == 2 and
+                        npcUtil.popFromQM(player, npc, { quadavOffset, quadavOffset + 1, quadavOffset + 2, quadavOffset + 3 }, { claim = true, hide = 0 })
+                    then
+                        return quest:messageSpecial(beadeauxID.text.TAKEN_FROM_YOU, 0, xi.ki.GLITTERING_FRAGMENT)
+                    elseif
+                        quest:getVar(player, 'Prog') == 2 and
+                        quest:getVar(player, 'magnes_nmKilled') == 1 and
+                        quest:getVar(player, 'nickel_nmKilled') == 1
+                    then
+                        return quest:progressEvent(124)
+                    end
+                end,
```

Comment:
```lua
                    if quest:getVar(player, 'Prog') == 2 then
                        if npcUtil.popFromQM(player, npc, { quadavOffset, quadavOffset + 1, quadavOffset + 2, quadavOffset + 3 }, { claim = true, hide = 0 }) then
                            return quest:messageSpecial(beadeauxID.text.TAKEN_FROM_YOU, 0, xi.ki.GLITTERING_FRAGMENT)
                        elseif
                            quest:getVar(player, 'magnes_nmKilled') == 1 and
                            quest:getVar(player, 'nickel_nmKilled') == 1
                        then
                            return quest:progressEvent(124)
                        end
                    end
                end,
```

Also, if the message states the KI was taken from you, shouldnt the KI actually be taken from you?